process.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "path/filepath"
+    "io/ioutil"
+    "strings"
+    "strconv"
+    "sync"
+)
+
+func main() {
+    args := os.Args
+    sourceDir, err := filepath.Abs(args[1])
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
```

Comment:
Sorry, I forgot that there is a function `log.Fatal`, it's more convenient to just call `log.Fatal(err)` and then it'll call `os.Exit` automatically